Products/zms/plugins/www/bootstrap/plugin/bootstrap.plugin.zmi.js

```diff
@@ -1290,6 +1290,9 @@ ZMIObjectTree.prototype.addPages = function(nodes) {
 		var css = [ node.is_page ? 'is_page' : 'is_page_element' ];
 		var callback = that.p['toggleClick.callback'];
 
+		if ((node.titlealt.toUpperCase().search('REDIRECT')>-1) &&
+			(node.attr_dc_identifier_url_redirect.trim() != '')) return;
```

Comment:
Is attr_dc_identifier_url_redirect in default ZMS...?!
-> optional chaining - handle undefined

Reply:
DONE in https://github.com/zms-publishing/ZMS/commit/2b82d37246821583d191874affbe1e98e2a9eb83.